core/app/sorters/spree/products/sort.rb

```diff
@@ -0,0 +1,43 @@
+module Spree
+  module Products
+    class Sort
+      def initialize(scope, params)
+        @scope = scope
+
+        @desc_order = String(params[:sort])[0] == '-'
+        @sort_field = @desc_order ? params[:sort][1..-1] : params[:sort]
+      end
+
+      def call
+        products = sort_by_updated_at(scope)
+        products = sort_by_price(products)
+
+        products
+      end
+
+      private
+
+      attr_reader :desc_order, :sort_field, :scope
+
+      def updated_at?
+        sort_field.present? && sort_field == 'updated_at'
+      end
+
+      def price?
+        sort_field.present? && sort_field == 'price'
+      end
+
+      def sort_by_updated_at(products)
+        return products unless updated_at?
+
+        products.order(Arel.sql("updated_at #{'DESC' if desc_order}"))
```

Comment:
What do you think about moving `Arel.sql(...)` to a separate method for better readability and flexibility?